examples/AspNetCore/README.md

```diff
@@ -13,6 +13,43 @@ service name, version and the machine on which this program is running.
 The sample rate is set to emit all the traces using `AlwaysOnSampler`.
 You can try out different samplers like `TraceIdRatioBasedSampler`.
 
+## How to run the example
```

Comment:
this is too complex. I expect it to be as simply as "open localhost:5000/weatherforecast" in your browser.

Reply:
5000 wouldn't work. It is a randomly picked number.

Reply:
why dont we make it a fixed one, instead of random ports?

Reply:
That's a good idea. I didn't think it was a right thing to do because it's in an auto generated file normally ignored by [.gitignore config](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/.gitignore#L60).

Reply:
I've updated it in the branch, which is not synced in the PR probably because it's closed.